### PR TITLE
RunSqlCommand: Use VarDumper when available

### PR DESCRIPTION
**Pull request by nicolas-grekas**
VarDumper output is so much prettier than var_dump's ;)

### CODE REVIEW

**Comment by Ocramius** [comment]
Same as doctrine/doctrine2#1491
No-go, sorry :-\

File: lib/Doctrine/DBAL/Tools/Console/Command/RunSqlCommand.php [hunk 1]
```diff
@@ -24,6 +24,8 @@
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Output\OutputInterface;
 use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\VarDumper\Dumper\CliDumper;
```
Comment (guilhermeblanco) [inline lib/Doctrine/DBAL/Tools/Console/Command/RunSqlCommand.php]: IIRC this cannot be used here or it'll crash

File: lib/Doctrine/DBAL/Tools/Console/Command/RunSqlCommand.php [hunk 2]
```diff
@@ -79,10 +81,22 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $resultSet = $conn->executeUpdate($sql);
         }
 
-        ob_start();
-        \Doctrine\Common\Util\Debug::dump($resultSet, (int) $depth);
-        $message = ob_get_clean();
+        if (class_exists('Symfony\Component\VarDumper\Dumper\CliDumper')) {
+            $cloner = new VarCloner();
```
Comment (guilhermeblanco) [inline lib/Doctrine/DBAL/Tools/Console/Command/RunSqlCommand.php]: Use FQCN here

File: lib/Doctrine/DBAL/Tools/Console/Command/RunSqlCommand.php [hunk 3]
```diff
@@ -79,10 +81,22 @@ protected function execute(InputInterface $input, OutputInterface $output)
...
+            $data = $cloner->cloneVar($resultSet)->withMaxDepth((int) $depth);
 
-        $output->write($message);
+            $dumper = new CliDumper();
```
Comment (guilhermeblanco) [inline lib/Doctrine/DBAL/Tools/Console/Command/RunSqlCommand.php]: And here

File: lib/Doctrine/DBAL/Tools/Console/Command/RunSqlCommand.php (hunk 1 above)
Comment (nicolas-grekas) [inline lib/Doctrine/DBAL/Tools/Console/Command/RunSqlCommand.php]: not sure to understand why?

**Comment by Ocramius** [comment]
@guilhermeblanco it's closed anyway

**Pull request closed without merging**